docs/rag/adrs/granite-embeddings.md

```diff
@@ -0,0 +1,39 @@
+- There are many other open source models of comparable size to the Granite embedding models.  However, most highly competitive models use MS-MARCO or other sources with problematic provenance.  Furthermore, IBM has [published benchmark results](https://www.ibm.com/new/announcements/ibm-granite-3-1-powerful-performance-long-context-and-more#Granite+embedding+models) showing that Granite's accuracy is highly competitive with other popular open source options of comparable size.
+
+## Decision
+
+InstructLab RAG will use [IBM Granite-Embedding-125m-English](https://huggingface.co/ibm-granite/granite-embedding-125m-english) as the default embedding model.
+
+## Consequences
+
+InstructLab is already using IBM Granite generative models as default options for model training.  Using an IBM Granite embedding model extends both the advantages and disadvantages of depending on models from IBM.  An advantage is the fact that IBM has an outstanding, well-earned reputation for respecting intellectual property rights and careful compliance with legal restrictions.  Users who value this trait are disproportionately likely to be using InstructLab already and are thus disproportionately likely to appreciate this default value.  A potential disadvantage is the fact that selecting another IBM model in addition to the ones that InstructLab is already using (for text generation and document conversion) could be perceived as prioritizing IBM's business strategy over the needs of this open source community project.  This ADR tries to mitigate that potential disadvantage by describing clear considerations that are well aligned with the needs of the user base and providing a precise argument for why IBM Granite embedding models are particularly well aligned with those criteria.
```

Comment:
In keeping with the spirit of the template of making things easy to scan and read, can we reorganize this into a bulleted list? One example might be:

```
* We will have consistency with usage of IBM models when possible.
* We should consider strategies of offsetting perception of prioritizing IBM models over the needs of the product in addition to the clear motivation described here, such as in documentation.
* Users will be able to rely on IBM's outstanding, well-earned reputation for respecting intellectual property rights and careful compliance with legal restrictions.
```